### PR TITLE
Adds BBQ sauce, and BBQ Ribs.

### PR DESCRIPTION
**Pull request by ArcaneMusic**
## About The Pull Request

With the end of summer rapidly passing us by, and the recent advent of the GRILL approaching, I felt like this was long overdue.
This introduces BBQ sauce, a new condiment created with 1 part ash, 1 part tomato juice, 1 part black pepper, and 3 parts saline glucose. You can add BBQ sauce like any other condiment, or you can use it to make the new BBQ ribs, made with BBQ sauce, 2 rods ("bones"), and 2 steaks.
BBQ and Ribs
(Left to right, Beaker with BBQ Sauce, BBQ Ribs, BBQ Sauce Packet, and the BBQ Sauce Condiment bottle.)
## Why It's Good For The Game

Thought it'd be thematic and fun to go along with #46844 while opening up the possibility for other BBQ style recipes.

## Changelog
:cl:
add: Added a new condiment, BBQ sauce.
add: Added a new recipe to make BBQ Ribs.
/:cl:

### CODE REVIEW

**Comment by carshalash** [comment]
Where the fuck is the vinegar?

**Comment by ArcaneMusic** [comment]
> Where the fuck is the vinegar?

More like WHAT the fuck is vinegar 😎

**Comment by carshalash** [comment]
brb making a code bounty.

**Comment by Tlaltecuhtli** [comment]
make the ribs make your hands dirty ( bloody overlay)

File: code/modules/food_and_drinks/recipes/food_mixtures.dm [hunk 1]
```diff
@@ -184,3 +184,9 @@
 	results = list(/datum/reagent/consumable/nutriment/peptides = 0.5)
 	required_reagents = list(/datum/reagent/consumable/nutriment/ = 0.5)
 	required_catalysts = list(/datum/reagent/medicine/metafactor = 0.5)
+
+/datum/chemical_reaction/bbqsauce
+	name = "BBQ Sauce"
+	id = /datum/reagent/consumable/bbqsauce
+	results = list(/datum/reagent/consumable/bbqsauce = 20)
+	required_reagents = list(/datum/reagent/consumable/sugar = 5, /datum/reagent/ash = 5, /datum/reagent/consumable/capsaicin = 5, /datum/reagent/water = 5)
```
Comment (81Denton) [inline code/modules/food_and_drinks/recipes/food_mixtures.dm]: Can't believe I'm bikeshedding about BBQ Sauce in a 2D clown abuse simulator, but..
Can you add salt and black pepper to the recipe? The lack of vinegar hurts me, but we just don't have that reagent in the codebase. Also, what's the ash for?

Comment (carshalash) [reply code/modules/food_and_drinks/recipes/food_mixtures.dm]: Smokiness silly.

Comment (81Denton) [reply code/modules/food_and_drinks/recipes/food_mixtures.dm]: Ah yeah that works

Comment (ArcaneMusic) [reply code/modules/food_and_drinks/recipes/food_mixtures.dm]: More than doable, sure.

File: code/modules/food_and_drinks/recipes/tablecraft/recipes_meat.dm [hunk 2]
```diff
@@ -198,3 +198,12 @@
 	)
 	result = /obj/item/reagent_containers/food/snacks/salad/ricepork
 	subcategory = CAT_MEAT
+
+/datum/crafting_recipe/food/ribs
+	name = "BBQ Ribs"
+	reqs = list(
+		/datum/reagent/consumable/bbqsauce = 5,
+		/obj/item/reagent_containers/food/snacks/meat/steak/plain = 2
```
Comment (Fikou) [inline code/modules/food_and_drinks/recipes/tablecraft/recipes_meat.dm]: would be funny if it required bones

**Comment by Grazyn** [comment]
No McRib?

**Comment by dr-bee** [comment]
Add tomato juice to the sauce recipe and you could call it a St. Louis style sauce, which are tomato instead of vinegar based.

**Comment by Tlaltecuhtli** [comment]
turn sauce into a component

**Comment by ArcaneMusic** [comment]
Ha, the moral of the story kids is compile your code before you sleep,
or don't sleep

File: code/modules/food_and_drinks/recipes/tablecraft/recipes_meat.dm (hunk 2 above)
Comment (ArcaneMusic) [reply code/modules/food_and_drinks/recipes/tablecraft/recipes_meat.dm]: Yeah, but lets be real, then nobody would ever make it.

Comment (Fikou) [reply code/modules/food_and_drinks/recipes/tablecraft/recipes_meat.dm]: @ArcaneMusic make it require rods as fake bones

File: code/modules/reagents/chemistry/reagents/food_reagents.dm [hunk 3]
```diff
@@ -748,4 +748,13 @@
...
 	if(prob(10))
 		M.say(pick("I hate my wife.", "I just want to grill for God's sake.", "I wish I could just go on my lawnmower and cut the grass.", "Yep, Quake. That was a good game...", "Yeah, my PDA has wi-fi. A wife I hate."), forced = /datum/reagent/consumable/char)
-	..()
\ No newline at end of file
+	..()
+
+
+/datum/reagent/consumable/bbqsauce
+	name = "BBQ Sauce"
+	description = "Sweey, Smokey, Savory, and gets everywhere. Perfect for Grilling."
```
Comment (Fikou) [inline code/modules/reagents/chemistry/reagents/food_reagents.dm]: "Sweey"

Comment (ArcaneMusic) [reply code/modules/reagents/chemistry/reagents/food_reagents.dm]: shoot

File: code/modules/food_and_drinks/recipes/tablecraft/recipes_meat.dm (hunk 2 above)
Comment (ArcaneMusic) [reply code/modules/food_and_drinks/recipes/tablecraft/recipes_meat.dm]: Done

**Comment by Farquaar** [comment]
> 
> 
> Add tomato juice to the sauce recipe and you could call it a St. Louis style sauce, which are tomato instead of vinegar based.
The also make a delicious white barbecue sauce with a mayonnaise base in Alabama.

**Comment by spessbandit** [comment]
>barbecue sauce
>mayonnaise
>MAYONNAISE

get the fuck out of this planet

**Comment by Farquaar** [comment]
> 
> 
> > barbecue sauce
> > mayonnaise
> > MAYONNAISE
> 
> get the fuck out of this planet

**Comment by nero1024** [comment]
Can I apply BBQ sauce to footwear to make BBQ Shoes?

**Comment by Anonymous583** [comment]
> Can I apply BBQ sauce to footwear to make BBQ Shoes?

fetish content

**Comment by nero1024** [comment]
> > Can I apply BBQ sauce to footwear to make BBQ Shoes?
> 
> fetish content

I want my meme streamer references.

**Comment by ArcaneMusic** [comment]
Alright, this should be ready.

**Pull request merged**